### YamlMime:ManagedReference
items:
- uid: System.IO.IsolatedStorage.IsolatedStorageException
  id: IsolatedStorageException
  children:
  - System.IO.IsolatedStorage.IsolatedStorageException.#ctor
  - System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.String)
  - System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: IsolatedStorageException
  nameWithType: IsolatedStorageException
  fullName: System.IO.IsolatedStorage.IsolatedStorageException
  type: Class
  summary: "隔離儲存區中的作業失敗時擲回例外狀況。"
  remarks: "`IsolatedStorageException`會使用 HRESULT ISS_E_ISOSTORE，0x80131450 的值。       最常見的隔離儲存區例外狀況的範例如下所示。      -遺漏的辨識項。 隔離儲存區需要辨識項 （組件和其來源的相關資訊），才能判斷程式碼的身分識別，並將它連接至正確相關聯的檔案空間。 如果沒有此資訊，無法使用隔離儲存區。      -無效的作業。 某些<xref:System.IO.FileStream>隔離儲存區不支援的作業。</xref:System.IO.FileStream>       如需執行個體的初始屬性值的清單`IsolatedStorageException`，請參閱<xref:System.IO.IsolatedStorage.IsolatedStorageException.%23ctor%2A>建構函式。</xref:System.IO.IsolatedStorage.IsolatedStorageException.%23ctor%2A>"
  syntax:
    content: >-
      [System.Runtime.InteropServices.ComVisible(true)]

      public class IsolatedStorageException : Exception
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor
  id: '#ctor'
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  langs:
  - csharp
  name: IsolatedStorageException()
  nameWithType: IsolatedStorageException.IsolatedStorageException()
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException()
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.IO.IsolatedStorage
  summary: "初始化的新執行個體<xref href=&quot;System.IO.IsolatedStorage.IsolatedStorageException&quot;></xref>使用預設屬性的類別。"
  remarks: "下表顯示<xref:System.IO.IsolatedStorage.IsolatedStorageException>.</xref:System.IO.IsolatedStorage.IsolatedStorageException>的執行個體的初始屬性值      |屬性 |值 |  |--------------|-----------|  |<xref:System.Exception.InnerException%2A>|`null`.|  |<xref:System.Exception.Message%2A>|當地語系化的錯誤訊息字串。 |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public IsolatedStorageException ();
    parameters: []
  overload: System.IO.IsolatedStorage.IsolatedStorageException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  langs:
  - csharp
  name: IsolatedStorageException(String)
  nameWithType: IsolatedStorageException.IsolatedStorageException(String)
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException(String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.IO.IsolatedStorage
  summary: "初始化的新執行個體<xref href=&quot;System.IO.IsolatedStorage.IsolatedStorageException&quot;></xref>類別指定的錯誤訊息。"
  remarks: "下表顯示<xref:System.IO.IsolatedStorage.IsolatedStorageException>.</xref:System.IO.IsolatedStorage.IsolatedStorageException>的執行個體的初始屬性值      |屬性 |值 |  |--------------|-----------|  |<xref:System.Exception.InnerException%2A>|`null`.|  |<xref:System.Exception.Message%2A>|錯誤訊息字串。 |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public IsolatedStorageException (string message);
    parameters:
    - id: message
      type: System.String
      description: "解釋例外狀況的原因的錯誤訊息。"
  overload: System.IO.IsolatedStorage.IsolatedStorageException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  langs:
  - csharp
  name: IsolatedStorageException(SerializationInfo,StreamingContext)
  nameWithType: IsolatedStorageException.IsolatedStorageException(SerializationInfo,StreamingContext)
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.IO.IsolatedStorage
  summary: "初始化的新執行個體<xref href=&quot;System.IO.IsolatedStorage.IsolatedStorageException&quot;></xref>使用序列化資料的類別。"
  remarks: "重新建構所需的資料流上傳輸的例外狀況物件的還原序列化期間呼叫這個建構函式。"
  syntax:
    content: protected IsolatedStorageException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "保存序列化的物件資料的物件。"
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "關於來源或目的端內容資訊。"
  overload: System.IO.IsolatedStorage.IsolatedStorageException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  langs:
  - csharp
  name: IsolatedStorageException(String,Exception)
  nameWithType: IsolatedStorageException.IsolatedStorageException(String,Exception)
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException(String,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.IO.IsolatedStorage
  summary: "初始化的新執行個體<xref href=&quot;System.IO.IsolatedStorage.IsolatedStorageException&quot;></xref>類別指定的錯誤訊息以及造成此例外狀況的內部例外狀況的參考。"
  remarks: "做為前一個例外狀況直接結果應該包括在前一個例外狀況的參考就會擲回的例外狀況<xref:System.Exception.InnerException%2A>屬性。</xref:System.Exception.InnerException%2A> <xref:System.Exception.InnerException%2A>屬性會傳回相同的值傳遞到建構函式，或`null`如果<xref:System.Exception.InnerException%2A>屬性沒有提供內部例外值給建構函式。</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       下表顯示<xref:System.IO.IsolatedStorage.IsolatedStorageException>.</xref:System.IO.IsolatedStorage.IsolatedStorageException>的執行個體的初始屬性值      |屬性 |值 |  |--------------|-----------|  |<xref:System.Exception.InnerException%2A>|內部例外狀況參考。 |  |<xref:System.Exception.Message%2A>|錯誤訊息字串。 |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public IsolatedStorageException (string message, Exception inner);
    parameters:
    - id: message
      type: System.String
      description: "解釋例外狀況的原因的錯誤訊息。"
    - id: inner
      type: System.Exception
      description: "造成目前例外狀況的例外狀況。 如果`inner`參數不是<xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;></xref>中, 引發目前的例外狀況<xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;></xref>處理內部例外狀況區塊。"
  overload: System.IO.IsolatedStorage.IsolatedStorageException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  isExternal: false
  name: IsolatedStorageException()
  nameWithType: IsolatedStorageException.IsolatedStorageException()
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException()
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.String)
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  isExternal: false
  name: IsolatedStorageException(String)
  nameWithType: IsolatedStorageException.IsolatedStorageException(String)
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  isExternal: false
  name: IsolatedStorageException(SerializationInfo,StreamingContext)
  nameWithType: IsolatedStorageException.IsolatedStorageException(SerializationInfo,StreamingContext)
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor(System.String,System.Exception)
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  isExternal: false
  name: IsolatedStorageException(String,Exception)
  nameWithType: IsolatedStorageException.IsolatedStorageException(String,Exception)
  fullName: System.IO.IsolatedStorage.IsolatedStorageException.IsolatedStorageException(String,Exception)
- uid: System.IO.IsolatedStorage.IsolatedStorageException.#ctor*
  parent: System.IO.IsolatedStorage.IsolatedStorageException
  isExternal: false
  name: IsolatedStorageException
  nameWithType: IsolatedStorageException.IsolatedStorageException
